canonical/canonical.com
# WD-3178 update URLs

## Done

Update items according to [Jira ticket](https://warthogs.atlassian.net/browse/WD-3178)
## QA

- Check out this feature branch

Update URLs of the following pages as follows:

/careers/company-culture/progression

/careers/company-culture/sustainability

/careers/company-culture/remote-work

/careers/company-culture/diversity


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3178
